guides/release/reference/accessibility-guide.md

```diff
@@ -1,128 +0,0 @@
-The purpose of this guide is to give every Ember developer a starting point for making your Ember application minimally accessible. As accessibility can be a complex issue, it should be noted that this guide only covers some of the basics, and should not be considered a comprehensive guide. A good place to start is [Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/) provided by the Web Accessibility Initiative (WAI).
```

Comment:
This file needs to have its contents replaced by a redirect. Otherwise people will get a 404 when switching between versions and it will be harder to find this kind of content in search engines. Search the word "redirect" and you will see many frontmatter examples.